### ISSUE TITLE
not loading any data iPhone 4 on iOS7.1.2

### ISSUE THREAD

**Issue by rainforest1155** [bryceco/GoMap#93]
I just updated the app to the latest version (hadn't used it in a while) and opening it, I get lots of errors about hostnames not found and things like that. The app isn't loading any map data and all I see is the usual user interface with spinner (if Bing layers are enabled, there's also the bing logo in the upper left).

I tried deleting and reinstalling the app, but it's still the same.

**Comment by bryceco** [comment]
The openstreetmap.com server was down for a while today. You might want to try again.

**Comment by rainforest1155** [comment]
I tried it again and am still getting "data not available" errors. Enabling aerial maps, I also still get the hostname not found errors.
Accessing OSM via a browser, I can see maps loading up fine.

In case there's still an issue with the OSM servers, I'll try again later and let you know.

**Comment by bryceco** [comment]
If it still isn't working I would delete and reinstall.

**Comment by rainforest1155** [comment]
I tried deleting the app and reinstalling before and it was still the same. The app was unable to load any map data, neither OSM nor Bing data.
Turns out that I had cellular data disabled for the app and while that shouldn't block the app from using WiFi, that seems to be the case. 
For most users, this won't be an issue, but some users may want to use the data from a mobile hotspot instead of their carrier plan, especially if you're in a foreign country. So it would be useful if the app didn't refuse to work if cellular data on the device is turned off.
